cg/apps/tb/api.py

```diff
@@ -75,7 +75,7 @@ def query_trailblazer(
         return ReadStream.get_content_from_stream(file_format=FileFormat.JSON, stream=response.text)
 
     def get_latest_completed_analysis(self, case_id: str) -> TrailblazerAnalysis | None:
```

Comment:
Is this only used for uploads?

Reply:
I think so it is used in:
`services/slurm_upload_service/slurm_upload_service.py`
`services/analysis_service/analysis_service.py`
`apps/tb/api.py`